docker-compose/monitor/Makefile

```diff
@@ -0,0 +1,50 @@
+.PHONY: build
+build: clean-jaeger
+	cd ../../ && \
+	make build-all-in-one && \
+	make docker-images-jaeger-backend && \
+	docker tag jaegertracing/all-in-one:latest jaegertracing/all-in-one:dev
```

Comment:
what's the reason for aliasing latest tag to dev tag?

Reply:
Ah, I initially aliased the tag because I noticed `docker rmi -f ...:latest` wasn't cleaning things up properly. I was still getting an older version.

However, I tested it again and seems to be working fine now. I _suspect_ because I'd switched over to using `docker system prune -f`.

I like the explicitness of a `latest` (for users that just want to demo SPM) vs `dev` (for users who are developers of SPM) version and this explicitness is mirrored in make targets, which seems nice as well (`make run` vs `make run-dev`).

Thoughts?